rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/rmnode/RMNodeImpl.java

```diff
@@ -92,14 +89,12 @@
     /** ProActive Node Object of the RMNode */
     private Node node;
 
-    /** URL of the node, considered as its unique ID */
-    private String nodeURL;
-
-    /** Name of the node */
-    private String nodeName;
-
     /** {@link VirtualNode} name of the node */
-    private String vnodeName;
+    // TODO: Investigate if this field is still required
```

Comment:
Shouldn't we create an issue instead of letting a comment for that kind of question ?